viewer-prototype/src/browser/trace-viewer/components/timegraph-output-component.tsx

```diff
@@ -326,7 +326,7 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
             }
             style = this.styleMap.get(value);
             if (!style) {
-                style = backupStyles[(value % backupStyles.length)];
+                style = backupStyles[(Math.abs(value) % backupStyles.length)];
                 this.styleMap.set(value, style);
             }
             return {
```

Comment:
I noticed a run time error because 'style' object can be undefined because for few traces, the 'value' could be negative. Hence we need to make sure we always get a valid 'style' object.

I couldn't reproduce this with any of the traces frequently. But saw this error a few times.

Reply:
I was able to reproduce it after several attempts. Below is the error 

timegraph-output-component.tsx:333 Uncaught (in promise) TypeError: Cannot read property 'color' of undefined
    at TimegraphOutputComponent.getDefaultElementStyle (timegraph-output-component.tsx:333)
    at TimegraphOutputComponent.getElementStyle (timegraph-output-component.tsx:269)
    at Object.rowElementStyleProvider (timegraph-output-component.tsx:58)
    at TimeGraphChart.push.../node_modules/timeline-chart/lib/layer/time-graph-chart.js.TimeGraphChart.createNewRowElement (time-graph-chart.ts:320)
    at time-graph-chart.ts:300

Nevertheless, its good to have this check.

Reply:
Thanks. Do you know the workflow which reproduces this consistently? Or you found it in a random workflow?

Reply:
It was random while navigating states.